Sharppy/sharppy_analysis.md

```diff
@@ -0,0 +1,380 @@
+sharppy/viz/hodo.py:23: unused attribute 'first' (60% confidence)
+sharppy/viz/hodo.py:241: unused function 'hodo_to_pix' (60% confidence)
+sharppy/viz/hodo.py:337: unused attribute 'center_loc' (60% confidence)
+sharppy/viz/hodo.py:519: unused attribute 'center_loc' (60% confidence)
+sharppy/viz/hodo.py:531: unused attribute 'center_loc' (60% confidence)
+sharppy/viz/hodo.py:541: unused attribute 'center_loc' (60% confidence)
+sharppy/viz/hodo.py:571: unused function 'mousePressEvent' (60% confidence)
+sharppy/viz/hodo.py:596: unused function 'mouseReleaseEvent' (60% confidence)
+sharppy/viz/hodo.py:734: unused function 'calculateStormMode' (60% confidence)
+sharppy/viz/hodo.py:785: unused function 'mouseMoveEvent' (60% confidence)
+sharppy/viz/map.py:32: unused function 'getPhi0' (60% confidence)
+sharppy/viz/map.py:207: unused variable 'south' (60% confidence)
+sharppy/viz/map.py:208: unused variable 'north' (60% confidence)
+sharppy/viz/map.py:213: unused variable 'typ' (60% confidence)
+sharppy/viz/map.py:277: unused attribute 'map_rot' (60% confidence)
+sharppy/viz/map.py:504: unused variable 'clicked_id' (60% confidence)
+sharppy/viz/map.py:511: unused variable 'clicked_id' (60% confidence)
+sharppy/viz/map.py:544: unused function 'mousePressEvent' (60% confidence)
+sharppy/viz/map.py:547: unused function 'mouseMoveEvent' (60% confidence)
+sharppy/viz/map.py:556: unused variable 'is_invertible' (60% confidence)
+sharppy/viz/map.py:561: unused function 'mouseReleaseEvent' (60% confidence)
+sharppy/viz/map.py:583: unused function 'mouseDoubleClickEvent' (60% confidence)
+sharppy/viz/map.py:584: unused variable 'is_invertible' (60% confidence)
+sharppy/viz/skew.py:156: unused function 'draw_moist_adiabat' (60% confidence)
+sharppy/viz/skew.py:337: unused attribute 'dp' (60% confidence)
+sharppy/viz/skew.py:560: unused function 'mouseReleaseEvent' (60% confidence)
+sharppy/viz/skew.py:589: unused function 'mousePressEvent' (60% confidence)
+sharppy/viz/skew.py:640: unused function 'mouseMoveEvent' (60% confidence)
+sharppy/viz/skew.py:1078: unused variable 'text_offset' (60% confidence)
+sharppy/viz/SPCWindow.py:59: unused attribute 'mode' (60% confidence)
+sharppy/viz/SPCWindow.py:81: unused attribute 'swap_inset' (60% confidence)
+sharppy/viz/SPCWindow.py:729: unused function 'keyPressEvent' (60% confidence)
+sharppy/viz/thermo.py:148: unused function 'mouseDoubleClickEvent' (60% confidence)
+sharppy/viz/thermo.py:593: unused function 'mousePressEvent' (60% confidence)
+sharppy/viz/vrot.py:274: unused function 'mouseDoubleClickEvent' (60% confidence)
+utils/async.py:81: unused variable 'prio' (60% confidence)
+utils/frozenutils.py:15: unused function 'freezeSupport' (60% confidence)
+```
+
+#### True Positives
+
+After inspecting the source code detected by Vulture as dead, we confirmed that
+the following parts indeed aren't being used anywhere and can therefore be
+deleted safely:
```

Comment:
Let's phrase this a little differently:

```
#### Probably dead code

After inspecting the source code that Vulture reports as unused, we believe that
the following parts indeed aren't used anywhere and could therefore be deleted:
```

Reply:
Thanks! The previous one was rather dingy.